doc/modules/metrics.rst

```diff
@@ -73,7 +73,7 @@ their cosine similarity :math:`k` is defined as:
 
 .. math::
 
-    k(x, y) = \frac{x y^\top}{\|x\| \|y\|}
+    k(x, y) = \frac{x y^    op}{\|x\| \|y\|}
```

Comment:
@marcelobeckmann this change is probably not intended (\t being replaced with a tab character). It happens multiple times throughout this file.